### ISSUE TITLE
Assign modifier key to extra mouse buttons [enhancement]

### ISSUE THREAD

**Issue by neddy** [noah-nuebling/Mac-Mouse-Fix#1069]
## Description
It would be great if there was the ability to assign modifier keys such as SHIFT, OPTION or COMMAND to a mouse button. At the moment I see that you can assign SHIFT ANOTHER key, but not just SHIFT.

## Rationale
Logitech have this option, I use it in games so that I can hold a mouse button to simulate holding the SHIFT key so I can bind things to SHIFT A or SHIFT B etc... I normally use this with the extra mouse buttons on the side of the mouse.

I would have bought the app if it had this function. At the moment my only option is to buy another Logitech mouse. 

Cheers

## Additional info
–

**Comment by amadav** [comment]
What version of the app are you using? I can set cmd + button for my mouse. Currently using 3.0.2

**Comment by neddy** [comment]
Yes, that is possible. However, this request is to be able to just set CMD or SHIFT without another key, so the mouse button can be used as a modifier key.

**Comment by Hsn37** [comment]
+1 on this. I want to use horizontal scroll by using Button4 as holding down the shift key.
